koku/masu/test/database/test_ocp_report_db_accessor.py

```diff
@@ -1032,3 +1034,52 @@ def test_populate_usage_costs_new_columns_no_report_period(self):
         with self.assertLogs("masu.database.ocp_report_db_accessor", level="INFO") as logger:
             self.accessor.populate_usage_costs("", "", start_date, end_date, self.provider_uuid)
             self.assertIn(expected, logger.output)
+
+    def test_populate_platform_and_worker_distributed_cost_sql_no_report_period(self):
+        """Test that updating monthly costs without a matching report period no longer throws an error"""
+        start_date = "2000-01-01"
+        end_date = "2000-02-01"
+        msg = "No report period for OCP provider, skipping platform_and_worker_distributed_cost_sql update."
+        context = {"provider_uuid": self.provider_uuid, "start_date": {start_date}}
+        expected = f"INFO:masu.database.ocp_report_db_accessor:{log_json(self.provider_uuid, msg, context)}"
+
+        with self.assertLogs("masu.database.ocp_report_db_accessor", level="INFO") as logger:
+            self.accessor.populate_platform_and_worker_distributed_cost_sql(
+                start_date, end_date, self.provider_uuid, {"platform_cost": True}
+            )
+            self.assertIn(expected, logger.output)
+
+    @patch("masu.database.ocp_report_db_accessor.pkgutil.get_data")
+    @patch("masu.database.ocp_report_db_accessor.OCPReportDBAccessor._execute_raw_sql_query")
+    def test_populate_platform_and_worker_distributed_cost_sql_called(self, mock_sql_execute, mock_data_get):
+        """Test that the platform distribution is called."""
+        dh = DateHelper()
+        start_date = dh.this_month_start.date()
+        end_date = dh.this_month_end.date()
```

Comment:
This code is fine, but just as an FYI, the base test class now has a `dh` attribute, so you can totally use this if you wanted:
```suggestion
        start_date = self.dh.this_month_start.date()
        end_date = self.dh.this_month_end.date()
```